src/Microsoft.DotNet.XHarness.InstrumentationBase.Xunit/Microsoft.DotNet.XHarness.InstrumentationBase.Xunit.csproj

```
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <IsPackable>true</IsPackable>
+    <TargetFrameworks>netstandard2.1;netcoreapp3.1</TargetFrameworks>
```

If the project stays, it should probably be only `netstandard2.1`:
```suggestion
    <TargetFrameworks>netstandard2.1</TargetFrameworks>
```

---

Not sure if we can use msbuild.sdk.extras?

---

might be worth just doing `netstandard2.1;monoandroid10.0;xamarin.ios10`. Then packing it will make sure to copy all the files to the right place and each OS gets what it needs.